src/App.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
+import './pages/App.css';
+import Calculator from './components/calculator';
+import Navbar from './components/Navbar';
 import Home from './pages/Home';
 import Quote from './pages/Quote';
-import Calculator from './components/calculator';
-import Navbar from './pages/Navbar';
+import NotMatch from './pages/NotMatch';
 
 function App() {
   return (
     <>
       <Navbar />
-      <Routes>
-        <Route path="/quote" element={<Quote />} />
-        <Route path="/home" element={<Home />} />
-        <Route path="/calculator" element={<Calculator />} />
-      </Routes>
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route path="/calculator">
+          <Calculator />
+        </Route>
+        <Route path="/quote">
+          <Quote />
+        </Route>
+        <Route path="*">
+          <NotMatch />
+        </Route>
+      </Switch>
```

Comment:
- [ ] I think here you are using an old version from the switch since it is not found in react-router-dom so I highly recommend using BrowserRoutes this is a new update and it does the same as switch check this [article](https://www.javatpoint.com/browserrouter-in-react) for more information and make sure to update it to brwoserrouter 🔥💯🚀